kai/reactive_codeplanner/task_runner/dependency/api.py

```diff
@@ -1,8 +0,0 @@
-from dataclasses import dataclass
-
-from kai.reactive_codeplanner.task_manager.api import ValidationError
-
-
-@dataclass
-class DependencyValidationError(ValidationError):
```

Comment:
@fabianvf I don't see this used anywhere so I removed if I could get a second pair of eyes that would be great.